### PR TITLE
First pass at horizontal scrolling chips

### PR DESCRIPTION
**Pull request by eygraber**

### CODE REVIEW

File: library/src/main/java/com/isapp/chips/ChipsView.kt [hunk 1]
```diff
@@ -0,0 +1,205 @@
...
+    holder.delete?.setOnClickListener {
+      listener?.onChipDeleted(chips[holder.adapterPosition])
+    }
+  }
+
+  override fun getItemCount() = chips.size
+
+  override fun getItemViewType(position: Int): Int {
+    val chip = chips[position]
+    return if(!chip.deletable and chip.icon) {
```
Comment (colinmadere) [inline library/src/main/java/com/isapp/chips/ChipsView.kt:180]: "Not deletable and has icon" = `JUST_TEXT`?  Looks like it should be `ICON`?  Looks like maybe all these are mixed up?

Comment (eygraber) [reply library/src/main/java/com/isapp/chips/ChipsView.kt:180]: How odd. Not sure how that happened but it will certainly get fixed 😂

Comment (eygraber) [inline library/src/main/java/com/isapp/chips/ChipsView.kt]: How odd. Not sure how that happened but it will certainly get fixed 😂

Comment (eygraber) [reply library/src/main/java/com/isapp/chips/ChipsView.kt:180]: How does it look now?

Comment (colinmadere) [reply library/src/main/java/com/isapp/chips/ChipsView.kt:180]: 👍

**Review by colinmadere — APPROVED** [review]

**Pull request merged**